assets/packages/adminpanel/src/components/subcomponents/_questionsgroups.vue

```diff
@@ -231,99 +231,104 @@ export default {
+                            <div
+                                v-for="question in orderQuestions(questiongroup.questions)"
+                                v-bind:key="question.qid"
+                                v-bind:class="questionItemClasses(question)"
+                                data-toggle="tootltip"
+                                class="list-group-item question-question-list-item ls-flex-row align-itmes-flex-between"
                                 :title="question.question_flat"
                                 @dragenter="dragoverQuestion($event, question, questiongroup)"
                             >
-                                    <i 
+                                    <i
                                         v-if="!$store.state.surveyActiveState"
-                                        class="fa fa-bars margin-right bigIcons dragPointer question-question-list-item-drag" 
+                                        class="fa fa-bars margin-right bigIcons dragPointer question-question-list-item-drag"
                                         draggable="true"
-                                        @dragend="endDraggingQuestion($event, question)" 
+                                        @dragend="endDraggingQuestion($event, question)"
                                         @dragstart="startDraggingQuestion($event, question, questiongroup)"
                                         @click.stop.prevent="()=>false"
                                     >
                                         &nbsp;
                                     </i>
                                 <a
-                                    :href="question.link"  
-                                    class="col-12 pjax question-question-list-item-link display-as-container" 
-                                    @click.stop.prevent="openQuestion(question)" 
-                                > 
-                                    <span class="question_text_ellipsize" :style="{ width: itemWidth }">
-                                        [{{question.title}}] &rsaquo; {{ question.question_flat }} 
-                                    </span> 
+                                    :href="question.link"
+                                    class="col-12 pjax question-question-list-item-link display-as-container"
+                                    @click.stop.prevent="openQuestion(question)"
+                                >
+                                    <span v-if="question.hidden == 0" class="question_text_ellipsize" :style="{ width: itemWidth }">
+                                        [{{question.title}}] &rsaquo; {{ question.question_flat }}
+                                    </span>
+
+                                    <span v-if="question.hidden == 1" class="question_text_ellipsize question-hidden" :style="{ width: itemWidth }">
```

Comment:
This is rather verbose a simple :class="question.hidden == 1 ? 'question-hidden' : '' " would suffice